src/uct/cuda/gdr_copy/gdr_copy_md.c

```diff
@@ -42,8 +42,8 @@ static ucs_config_field_t uct_gdr_copy_md_config_table[] = {
      "shared across all contexts",
      ucs_offsetof(uct_gdr_copy_md_config_t, shared), UCS_CONFIG_TYPE_BOOL},
 
-    {"RCACHE", "try", "Enable using memory registration cache",
-     ucs_offsetof(uct_gdr_copy_md_config_t, enable_rcache), UCS_CONFIG_TYPE_TERNARY},
+    {"RCACHE", "yes", "Enable using memory registration cache",
+     ucs_offsetof(uct_gdr_copy_md_config_t, enable_rcache), UCS_CONFIG_TYPE_BOOL},
```

Comment:
Is it safe to change the option type from `ternary` to `bool`? How do we handle the case when the new version will be run with `RCACHE=try`? If there is a crash, then it is better to keep it `ternary`, but print a message that `try` is treated as `yes`, and will be removed.

Reply:
we haven't released a version with a ternary variant yet, so maybe it's not that big problem